src/core/Akka/Actor/Address.cs

```diff
@@ -299,11 +299,104 @@ public static Address Parse(string address)
             }
         }
 
+        /// <summary>
+        /// Parses a new <see cref="Address"/> from a given string
+        /// </summary>
+        /// <param name="span">The span of address to parse</param>
+        /// <param name="address">If <c>true</c>, the parsed <see cref="Address"/>. Otherwise <c>null</c>.</param>
+        /// <returns><c>true</c> if the <see cref="Address"/> could be parsed, <c>false</c> otherwise.</returns>
+        public static bool TryParse(ReadOnlySpan<char> span, out Address address)
```

Comment:
So where does this get used? I never wrote a `Span<T>`-ified method for `Address.Parse` because it usually would get parsed on its own via `ActorPath.Parse`.

Reply:
The ActorPath parsing block split for the `ActorPathCache` extension,
this made it possible and required to move the Address parsing code out.

Reply:
Ah ok, so if I understand correctly - `ActorPath.Parse` now calls `Address.Parse`, whereas it used to not do that before - correct?

Reply:
I found a small bug in the original code and extended it to support full paths parsing instead of a clean address string.
One new unit test is included. 

I needed to create multiple overloads with `string` args to not get the Span problem in the unit test project.

Reply:
LGTM